trucks.js

```diff
@@ -503,6 +519,102 @@ var foodTrucks = [
   }
 ];
 
+var trucks = (function(){
```

Comment:
while this works to wrap it in an iife, it's unnecessary in CommonJS/Node modules. Instead, just use the code that inside the iife in the file, but use `module.exports = ` instead of `return`